scripts/doc-generation/generate-maps-docs.sh

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+UTILS_PATH="$SCRIPT_DIR/../utils.sh"
+
+# shellcheck source=../utils.sh
+source "$UTILS_PATH"
```

Comment:
Getting a shellcheck warning here:

```
shellcheck: warning
note - Not following: ../utils.sh was not specified as input (see shellcheck -x). [SC1091]
```

Reply:
Cannot reproduce on either of the two MacBooks. Looks like `scripts/.shellcheckrc` didn't parse for some reason. Did you open a repository root project in your Visual Studio Code?

![image](https://user-images.githubusercontent.com/735178/144843286-e9b9a0ce-a8df-4673-b028-e29b300e7c5e.png)

Reply:
I'm using Sublime + https://github.com/SublimeLinter/SublimeLinter-shellcheck

The issue goes away if I open `scripts/doc-generation` folder.